charts/nitro/templates/tests/test.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "{{ include "nitro.fullname" . }}-ci-configmap"
+data:
+  config.yaml: |
+    {{- toYaml .Values.ci | nindent 4 }}
```

Comment:
does this mean the `ci.secrets` will also be present in this config map?

Reply:
great catch.

Reply:
You asked for a re-review but as far as I can see it still looks like the secrets would be added to this config map. Did you want to change that? Let me know if I am mistaken :)